README.md

```diff
@@ -47,4 +47,4 @@ soccer_et.getTeamDetailJSON('ኢትዮጵያ ቡና').then(detail => console.log
 - [x] Team details
 - [ ] Player details
 - [ ] Top players list
-- [ ] News
```

Comment:
There was no change here?

Reply:
It was probably an accidental white-space change

Reply:
Makes sense